control/statesp.py

```diff
@@ -1338,14 +1338,6 @@ def sample(self, Ts, method='zoh', alpha=None, prewarp_frequency=None,
         copy_names : bool, Optional
             If True, copy the names of the input signals, output
             signals, and states to the sampled system.
-
-        Returns
-        -------
-        sysd : StateSpace
-            Discrete-time system, with sampling rate Ts
-
-        Additional Parameters
-        ---------------------
```

Comment:
As above, this should be named "Other Parameters" and should go _after_ the "Returns" section.